.github/workflows/cargo_build.yml

```diff
@@ -18,17 +18,76 @@ permissions:
   contents: read
 
 jobs:
-  cargo-builds:
-    name: cargo_build/cargo-builds (bpr)
-    runs-on: ${{ matrix.os }}
+  parallel-pcc-cpu:
+    name: cargo_build/parallel-pcc-cpu
+    runs-on: large_ubuntu_16
+    strategy:
+      matrix:
+        batch: [ 1, 2, 3, 4, 5, 6, 7 ]
```

Comment:
we can do better and query the Makefile or grep the makefile to know how many batches there are and automate the matrix population (see ntt workflow for dynamic matrices) 😉

Reply:
if easier write the command directly in the Lakefile and have the output of the Makefile command be stored in the github output by the workflow

Reply:
Way more convenient this way and reduce maintenance issues if a new pcc recipe is created.